src/command/stamp/buy.ts

```diff
@@ -0,0 +1,35 @@
+import { LeafCommand, Option } from 'furious-commander'
+import { StampCommand } from './stamp-command'
+
+export class Buy extends StampCommand implements LeafCommand {
+  public readonly name = 'buy'
+
+  public readonly aliases = []
+
+  public readonly description = 'Buy postage stamp'
+
+  @Option({ key: 'depth', description: 'Depth of the postage stamp', type: 'number', required: true, minimum: 17 })
+  public depth!: number
+
+  @Option({ key: 'amount', description: 'Amount of the postage stamp', type: 'bigint', required: true, minimum: 1 })
+  public amount!: bigint
+
+  @Option({ key: 'label', description: 'Label of the postage stamp' })
```

Comment:
The descriptions could be clearer about the terms like
amount -> `epresents the value per chunk, which depreciates over time with new mined blocks.`